cmd/key-helper/importKey.go

```diff
@@ -56,11 +71,11 @@ func importKeyCommand() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			if err := importKeyToTransit(ctx, vaultClient, ciphertext, keyName, keyType); err != nil {
+			if err := importKeyToTransit(ctx, vaultClient, ciphertext, transitKeyName, keyType, transitEngineMountName); err != nil {
 				return err
 			}
 			fmt.Println("Successfully imported key to transit")
-			if err := importCertToKV(ctx, vaultClient, certPath, keyName); err != nil {
+			if err := importCertToKV(ctx, vaultClient, certPath, keyName, kvEngineMountName); err != nil {
```

Comment:
I realize this was before your time, but I am a little confused why we do both a KV and Transit import :-D

Reply:
Ahhh right, some older Transit engines don't support X509 chains.... Oh, OpenBao's doesn't because that feature was added in 1.15...? https://openbao.org/api-docs/secret/transit/